src/index.html

```diff
@@ -0,0 +1,363 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+<header class="main-header">
+    <div class="wrapper">
+        <nav class="main-navigation">
+            <a class="main-header-logo" href="#" >IC "Repair Design Project"</a>
+            <ul class="site-navigation">
+                <li class="site-navigation__item">
+                    <a href="#" class="site-navigation__link selected">home</a></li>
+                <li class="site-navigation__item">
+                    <a href="#" class="site-navigation__link">project</a></li>
+                <li class="site-navigation__item">
+                    <a href="#" class="site-navigation__link">measurement</a></li>
+                <li class="site-navigation__item">
+                    <a href="#" class="site-navigation__link">team</a></li>
+                <li class="site-navigation__item">
+                    <a href="#" class="site-navigation__link">reviews</a></li>
+                <li class="site-navigation__item">
+                    <a href="#" class="site-navigation__link">contacts</a></li>
+            </ul>
+            <ul class="main-navigation__contacts">
+                <li class="main-navigation__contacts__item"><a class="main-navigation__contacts__link" href="#">+7 (928) 768-32-39</a></li>
+                <li class="main-navigation__contacts__item"><button class="main-navigation__contacts__btn">Request a call</button></li>
+            </ul>
+        </nav>
+    </div>
+</header>
+
+<main class="page-main">
+
+    <section class="features wrapper">
+        <div class="features__contact">
+            <div class="instagram">
+                <a class="instagram__link" href="#">INSTAGRAM</a>
+            </div>
+        </div>
+        <div class="features__content">
+            <h1 class="features__content-title">Turnkey Repair in the <br> city of <span>Rostov-on-Don</span></h1>
```

Comment:
Approach with using <br> for line break is not a reliable one, because it makes a layout irresponsive. It's better to set width for the container